app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughTopicListFragmentTest.kt

```diff
@@ -92,6 +94,48 @@ class WalkthroughTopicListFragmentTest {
     )
   }
 
+  // If progressBar is getting hidden on scrolling to the second row of items or not
+  @Test
+  fun testWalkthroughWelcomeFragment_portrait_recyclerViewIndex4__progressBarIsHidden() {
+    launch<WalkthroughActivity>(createWalkthroughActivityIntent(0)).use {
+      onView(withId(R.id.walkthrough_welcome_next_button)).perform(scrollTo(), click())
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.walkthrough_topic_recycler_view)).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
+          4
+        )
+      )
+      onView(withId(R.id.walkthrough_progress_bar)).check(
+        matches(
+          withEffectiveVisibility(
+            Visibility.GONE
+          )
+        )
+      )
+    }
+  }
+
+  // If header text gets visible on scrolling to the second row of items or not
```

Comment:
Needs proper framing of this and above sentence.

Reply:
@rt4914 Same here